coq-unimath.opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "2.0"
+maintainer: "The UniMath Development Team"
+homepage: "https://github.com/UniMath/UniMath"
+dev-repo: "git+https://github.com/UniMath/UniMath.git"
+bug-reports: "https://github.com/UniMath/UniMath/issues"
+license: "Kind of MIT"
```

Comment:
What does "kind of MIT" mean?  It's probably better simply to say "see the file LICENSE.md"

Reply:
I would imagine that this file should be somewhat self-contained?

Reply:
Yes, I think it is meant to be self-contained.
Of course, we can change this description if we find something more appropriate.
We can also remove all the metadata (license, description etc), which is meaningful only inside a package collection.
However, it would be useful to keep here a complete opam file ready to be copied in the opam-coq-archive whenever we want to propose an update (e.g., for the future releases of the Coq Platform distribution).